packages/daemon/src/model/job.ts

```diff
@@ -134,6 +142,16 @@ export default async function model(sequelize: Sequelize): Promise<void> {
     },
     endTime: {
       type: INTEGER
+    },
+    params: {
+      type: STRING,
+      get() {
+        const rawParams: string = this.getDataValue.call(this, 'params');
```

Comment:
See https://www.sqlite.org/json1.html, shall we install this extension to support JSON?

Reply:
`sequelize` v6 support JSON for sqlite3, we could upgrade it from v5.

Reply:
The current solution (customized getter & setter) is caused by `sequelize` not `sqlite`

Reply:
As @FeelyChau said at #628, we could upgrade the sequelize.

Reply:
Sure, I think for this PR, we should remain the same. And the upgrade process should start on another PR.

Reply:
Shall we upgrade it firstly? Because this will change the database schema.

Reply:
IMO, upgrading for database schema would be tedious and time-consuming. And it requires to change db schema for the whole project. The db schema modification should not block a new feature.

Reply:
Done with the upgrade of sequelize.